### DISCUSSION TITLE
Why we have no access to deriveState on transaction endpoints?

### DISCUSSION THREAD

**Discussion by filipesoccol** [wevm/frog-archived#164] [Q&A] [answered]
I am working on a frame and using deriveState to pass informations forward. But apparently deriveState isn't accessible on TransactionContext. There is a reason for it?

How should I collect informations for send in a transaction?

**Comment by ggomaeng** [comment]
> I am working on a frame and using deriveState to pass informations forward. But apparently deriveState isn't accessible on TransactionContext. There is a reason for it?
> 
> How should I collect informations for send in a transaction?

Related: https://github.com/wevm/frog/discussions/163

**Comment by dalechyn** [comment, marked as answer]
State is not passed to transaction handlers. You can get `previousState` though.
